### PR TITLE
Rename My Files to My Works, fixes #1414

### LINKED ISSUE #1414: My Files on the listing should be My Works

### PR DESCRIPTION
**Pull request by awead**

### CODE REVIEW

File: app/views/my/index.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 <% @page_title = "#{current_tab.capitalize} listing" %>
 
-<h1 class="sr-only">My Files, Collections, Highlights, and Files Shared with Me</h1>
+<h1 class="sr-only">My Works, Collections, Highlights, and Files Shared with Me</h1>
```
Comment (mjgiarlo) [inline app/views/my/index.html.erb]: Should we also `s/Files Shared with Me/Works Shared with Me/g` or should that be a separate PR?

File: spec/helpers/batch_edits_helper_spec.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@
       @max_batch_size = 10
     end
 
-    context "with my files" do
+    context "with my works" do
       it "shows the check all dropdown" do
         allow(helper).to receive(:params).and_return(controller: "my/files")
```
Comment (mjgiarlo) [inline spec/helpers/batch_edits_helper_spec.rb]: The controller won't need to be "my/works" here?

File: spec/helpers/dashboard_helper_spec.rb [hunk 3]
```diff
@@ -38,11 +38,11 @@
...
 
   describe "#on_my_files" do
-    it "returns false when the controller isn't my files" do
+    it "returns false when the controller isn't my works" do
       allow(helper).to receive(:params).and_return(controller: "my/collections")
       expect(helper).to_not be_on_my_files
     end
-    it "returns true when the controller is my files" do
+    it "returns true when the controller is my works" do
       allow(helper).to receive(:params).and_return(controller: "my/files")
```
Comment (mjgiarlo) [inline spec/helpers/dashboard_helper_spec.rb]: The controller won't need to be "my/works" here?

File: spec/helpers/sufia_helper_spec.rb [hunk 4]
```diff
@@ -100,13 +100,13 @@
     end
 
     context "when the user is on the dashboard page" do
-      it "defaults to My Files" do
+      it "defaults to My Works" do
         allow(helper).to receive(:params).and_return(controller: "dashboard")
         expect(helper.search_form_action).to eq(sufia.dashboard_files_path)
```
Comment (mjgiarlo) [inline spec/helpers/sufia_helper_spec.rb]: OK, I see most of my comments boil down to "how far should this PR go?" Because I was wondering here whether `dashboard_files_path` should also be renamed to `dashboard_works_path`.

File: spec/routing/route_spec.rb [hunk 5]
```diff
@@ -78,7 +78,7 @@
       expect(get: '/dashboard/activity').to route_to(controller: 'dashboard', action: 'activity')
     end
 
-    it "routes to my files tab" do
+    it "routes to my works tab" do
       expect(get: '/dashboard/files').to route_to(controller: 'my/files', action: 'index')
```
Comment (mjgiarlo) [inline spec/routing/route_spec.rb]: See prior comments, all in the same vein.

**Comment by awead** [comment]
@mjgiarlo yeah, there's a big rabbit hole here... see #1431. My initial inclination is to keep the changes small, so go with this, but open additional tickets for the my controller issue, as well as pretty much *every* other instance of the word "files" in the code.

**Comment by mjgiarlo** [comment]
:rabbit2: :black_circle:

**Comment by awead** [comment]
@mjgiarlo I think the other stuff should get address in #1424 and #1431, which I'm working on now. Is there anything outstanding here you'd like to see changed?

File: config/locales/sufia.en.yml [hunk 6]
```diff
@@ -96,7 +96,7 @@ en:
         following:          "People you follow"
         followers:          "People who are following you"
       my:
-        files:        "My Files"
+        files:        "My Works"
```
Comment (grosscol) [inline config/locales/sufia.en.yml]: Should the key for this be "works:" ?  Or is that in another PR?

Comment (awead) [inline config/locales/sufia.en.yml]: Yeah, I'm working on that PR right now.
